core/metamodel/src/main/java/org/apache/isis/objectstore/jdo/metamodel/facets/prop/column/MaxLengthDerivedFromJdoColumnAnnotationFacetFactory.java

```diff
@@ -112,19 +113,21 @@ private void validate(ObjectSpecification objectSpec, ValidationFailures validat
                     return;
                 }
 
-                final List<ObjectAssociation> associations = objectSpec.getAssociations(Contributed.EXCLUDED, ObjectAssociation.Predicates.PROPERTIES);
-                for (ObjectAssociation association : associations) {
-
+                final Stream<ObjectAssociation> associations = objectSpec
+                        .streamAssociations(Contributed.EXCLUDED)
+                        .filter(ObjectAssociation.Predicates.PROPERTIES);
+                
+              //FIXME[ISIS-1976] change in behavior possible bug
```

Comment:
bug: should be 'continue' ... also affects 1.16.x
(there is a return in this loop, which should be 'continue' but correctly remains a 'return' now because we are inside of a lambda expression)